[Issue] create project via CLI (Windows 10)

### Details

### Problem

Application fails to start due to a missing native dependency: `@resvg/resvg-js-win32-x64-msvc`.

### Context

Occurs when running the backend/API service on Windows (`C:\workspace\project2`).
The error originates from `@resvg/resvg-js` and propagates through `@hexabot-ai/api` services during app initialization.

### Steps to Reproduce (OS: Windows 10)
1. `hexabot create project2`
2. `cd project2`
3. `hexabot start`
### Expected

Application should start successfully without missing module errors.

### Impact

Blocks local development and prevents the API service from starting on Windows environments.

### Extra

Error:

```bash
Error: Cannot find module '@resvg/resvg-js-win32-x64-msvc'
code: 'MODULE_NOT_FOUND'
```

Stack trace points to:

```bash
node_modules/@resvg/resvg-js/js-binding.js
```

<img width="1107" height="942" alt="Image" src="https://github.com/user-attachments/assets/7d072014-a946-4116-845a-101f519ef7cc" />

--- a/packages/api/package.json
+++ b/packages/api/package.json
@@ -179,7 +179,8 @@
   "optionalDependencies": {
     "@css-inline/css-inline-linux-arm64-musl": "^0.14.1",
     "@resvg/resvg-js-darwin-arm64": "^2.6.2",
-    "@resvg/resvg-js-linux-arm64-musl": "^2.6.2"
+    "@resvg/resvg-js-linux-arm64-musl": "^2.6.2",
+    "@resvg/resvg-js-win32-x64-msvc": "2.6.2"
   },
   "jest": {
     "globalSetup": "<rootDir>/../test/global-setup.ts",

